rift/fib_route.py

```diff
@@ -0,0 +1,26 @@
+import packet_common
+
+
+class FibRoute:
```

Comment:
@brunorijsman as you can see, FibRoute haven't got the concept of Owner (as we discussed)

Reply:
perfect, that is exactly what I had in mind